### ISSUE TITLE
how to reproduce the demo

### ISSUE THREAD

**Issue by bigsheep2018** [skyhehe123/SA-SSD#59]
hello sa-ssd team,
       Thanks for sharing this wonderful work. 
       May I know how can I reproduce the demo, because, by default,  the training set or validation set is a random ordered sequence.
       Or may i know where do you get the right ordered sequence on the kitti website.

**Comment by skyhehe123** [comment]
@bigsheep2018 . draw_lidar() and draw_gt_bboxes() in kitti_utils.py can help
